doc/introduction.rst

```diff
@@ -7,12 +7,81 @@ Introduction: nistats in a nutshell
     :depth: 1
 
 
-What is nistats: Bold-fMRI analysis
+What is nistats?
 ===========================================================================
 
-.. topic:: **Why use nistats?**
+.. topic:: **What is nistats?**
 
-    Nistats is a python module to perform voxel-wise univariate analyses of functional magnetic resonance images (fMRI), using general linear models. It provides functions to create design matrices, at the subject and group levels, to estimate them from images series and to compute statistical maps (contrasts)
+   Nistats is a python module to perform voxel-wise analyses of functional magnetic resonance images (fMRI) using linear models. It provides functions to create design matrices, at the subject and group levels, to estimate them from images series and to compute statistical maps (contrasts). It allows to perform the same statistical analyses as `a SPM`_ or `a FSL`_ (but it does not provide tools for preprocessing stages (realignement, spatial normalization, etc.); for this, see `a nipype`_.
+
+.. _a SPM: https://www.fil.ion.ucl.ac.uk/spm/
+
+.. _a FSL: https://www.fmrib.ox.ac.uk/fsl
+
+.. _a nipype: https://nipype.readthedocs.io/en/latest/>
+
+
+
+A primer on BOLD-fMRI data analysis
+===================================
+
+Functional magnetic resonance imaging (fMRI) is based on the fact that when local neural activity increases, increases in metabolism and blood flow lead to fluctuations of the relative concentrations of oxyhemoglobin (the red cells in the blood that carry oxygen) and deoxyhemoglobin (the same red cells after they have delivered the oxygen). Because oxy- and deoxy-hemoglobin have different magnetic properties (one is diamagnetic while the other is paramagnetic), they affect the local magnetic field in different ways. The signal picked up by the MRI scanner is sensitive to these modifications of the local magnetic field. To record cerebral activity, during functional sessions, the scanner is tuned to detect this "Blood Oxygen Level Dependent" (BOLD) signal.
+
+Brain activity is measured in sessions that span several minutes, while the participant performs some a cognitive task and the scanner acquires brain images, typically every 2 or 3 seconds (the time between two successive image acquisition is called the Repetition time, or RT).
+
+A cerebral MR image provides a 3D image of the brain that can be decomposed into `a voxels`_ (the equivalent of pixels, but in 3 dimensions). The series of images acquired during a functional session provides, in each voxel, a time series of positive real number representing the MRI signal, sampled at the RT.
+
+.. _a voxels: https://en.wikipedia.org/wiki/Voxel
+
+.. figure:: images/stimulation-time-diagram.png
```

Comment:
I feel figure here is not relevant. Relevant place would be below which is already there.